lib/index.js

```diff
@@ -99,10 +99,6 @@ Castle.prototype._identifyFromCache = function() {
   // See if there is a cached user, and call identify if so
   var userId = this.analytics.user().id();
   if (userId) {
-    window._castle.q.push(['setUserId', userId]);
-    var traits = this.analytics.user().traits();
-    if (traits) {
-      window._castle.q.push(['setUser', traits]);
-    }
+    window._castle.q.push(['identify', userId, this.analytics.user().traits()]);
```

Comment:
Since this is a quite critical path, have you sanity checked all combinations of null or empty string `id` and `traits` in a real environment?